src/Model.js

```diff
@@ -47,6 +78,10 @@ export default class Model extends StaticModel {
     return this[this.primaryKey()]
   }
 
+  updateMethod() {
+    return 'PUT'
+  }
```

Comment:
As discussed, this should be removed , in favor of new config() method.

Reply:
Yes, it's on TODO list.

Reply:
This PR requires the #142. The `updateMethod` will be removed there.